packages/core/lib/src/widgets/ui/navigation/app_drawer.dart

```diff
@@ -0,0 +1,143 @@
+import 'package:core/core.dart';
+import 'package:flutter/material.dart';
+
+/// App drawer widget
+class AppDrawer extends StatelessWidget {
+  /// Creates App drawer widget
+  const AppDrawer({
+    super.key,
+    required this.user,
+  });
+
+  /// The logged in user
+  final User user;
+
+  /// List of drawer menu items with localized labels
+  List<MenuListItemData> getDrawerMenuItems(BuildContext context) {
+    return [
+      MenuListItemData(
+        label: AppLocalizations.of(context)!.profile,
+        icon: TwitterIcons.user,
+      ),
+      MenuListItemData(
+        label: AppLocalizations.of(context)!.lists,
+        icon: TwitterIcons.list,
+      ),
+      MenuListItemData(
+        label: AppLocalizations.of(context)!.topics,
+        icon: TwitterIcons.topics,
+      ),
+      MenuListItemData(
+        label: AppLocalizations.of(context)!.notifications,
+        icon: TwitterIcons.bell,
+      ),
+      MenuListItemData(
+        label: AppLocalizations.of(context)!.bookmarks,
+        icon: TwitterIcons.bookmark,
+      ),
+    ];
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final listItems = getDrawerMenuItems(context);
+
+    return Container(
+      width: 320,
```

Comment:
are we modeling this with hard-coded width? `MediaQuery` might be good here.

Reply:
Yes we should use `MediaQuery` but I didn't use it here because it might not work properly in Widgetbook. Will try it with device frame as descussed.